src/main/java/com/exasol/adapter/dialects/mysql/MySqlGenerationVisitor.java

```diff
@@ -0,0 +1,31 @@
+package com.exasol.adapter.dialects.mysql;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.exasol.adapter.AdapterException;
+import com.exasol.adapter.dialects.*;
+import com.exasol.adapter.sql.*;
+
+public class MySqlGenerationVisitor extends SqlGenerationVisitor {
+    public MySqlGenerationVisitor(SqlDialect dialect, SqlGenerationContext context) {
```

Comment:
Since in the documentation it is named `MySqlSQLDialectIT`, would it makes to rename this class to `MySqlSqlGenerationVisitor`?

Reply:
Good finding. I'll fix it